package.json

```diff
@@ -7,7 +7,9 @@
   "license": "MIT",
   "files": [
     "src",
```

Comment:
I'm thinking src should not be included here actually...
And then there are some more things which could probably be removed from here as well.
```
"!**/__tests__",
    "!**/__fixtures__",
    "!**/__mocks__",
    "!**/node_modules",
    "!jest.config.js",
    "!babel.config.js",
    "!example/**"
```

But it doesn't have to be done in this PR.

Reply:
moblie maybe want src